background.js

```diff
@@ -202,3 +202,25 @@ chrome.storage.local.get('version', prefs => {
 		}
 	}
 });
+
+chrome.runtime.onInstalled.addListener(({reason, previousVersion, id} = {}) => {
+	// reason: install, update, chrome_update, shared_module_update
+	if (!/update/.test(reason)) {
+		return;
+  }
+  console.log(reason);
+	const contentScripts = chrome.app.getDetails().content_scripts;
+	chrome.windows.getAll({populate: true}, windows => {
+		windows.forEach(win => {
+			win.tabs.forEach(tab => {
+				contentScripts
+					.filter(cs => cs.matches.some(m => m == '<all_urls>' || tab.url.startsWith(m.replace('*', ''))))
```

Comment:
Just a note to remember we shouldn't convert `"matches": ["http://userstyles.org/*", "https://userstyles.org/*"],` to  `"matches": ["*://userstyles.org/*"],` in the `manifest.json` in the future.